source/reference/api/reverse.txt

```diff
@@ -63,16 +63,31 @@ Clusters that began with MongoDB 4.2 or older and were since
 upgraded may include unique indexes that are not properly formatted.
 
 To correct indexes, you can :ref:`resync <resync-replica-member>` all nodes
-in the original source cluster.  If you don't want to resync the cluster, you
-can use the :method:`db.collection.validate` method on each collection to 
-determine whether it contains improperly formatted unique indexes.
+in the original source cluster.  To resync all nodes: 
+
+.. procedure::
+   :style: normal
+
+   .. step:: Resync all secondaries, one by one. 
+
+      For a tutorial on resyncing nodes, see :ref:`Resync a Member of a
+      Replica Set <resync-replica-member>`. 
+
+   .. step:: Step down the primary node and step up one of the secondary nodes.
+   
+   .. step:: Resync the old primary node to the new primary node. 
+
+Alternatively, to avoid resyncing, you can use the
+:method:`db.collection.validate` method with ``full = false`` on each
+collection with unique indexes to determine whether each collection
+contains improperly formatted unique indexes. 
 
 .. code-block:: javascript
```

Comment:
[suggestion] This code block here seems a bit random and not super useful. We might want to remove this.

Reply:
Done!